src/main/telemetry/srxl.c

```diff
@@ -147,18 +150,31 @@ typedef struct
 */
 
 #define STRU_TELE_RPM_EMPTY_FIELDS_COUNT 8
+#define STRU_TELE_RPM_EMPTY_FIELDS_VALUE 0xff
+
+#define SPEKTRUM_RPM_UNUSED 0xffff
+#define SPEKTRUM_TEMP_UNUSED 0x7fff
 
 bool srxlFrameRpm(sbuf_t *dst, timeUs_t currentTimeUs)
 {
+
+    int16_t coreTemp = SPEKTRUM_TEMP_UNUSED;
+#if defined(USE_ADC_INTERNAL)
+    coreTemp = getCoreTemperatureCelsius();
+    coreTemp = coreTemp*9/5+32; // C -> F
```

Comment:
Sorry, I thought I'd mentioned that in my last review already, but there should be spaces between the operators here.

Reply:
OK, will fix.